Library/Homebrew/utils/tty.rb

```diff
@@ -59,7 +59,9 @@ def reset_escape_sequence!
   end
 
   def to_s
-    return "" unless $stdout.tty?
+    if ENV["HOMEBREW_NO_COLOR"] || !$stdout.tty?
```

Comment:
If this fits on one line <= 80 characters then you can do that otherwise: no worries as-is.